get_dependencies.py

```diff
@@ -3,15 +3,25 @@
 import tarfile
 from zipfile import ZipFile
 import requests
+from rfc6266 import parse_requests_response
 
+def download(url):
+    response = requests.get(url, stream=True)
+    total = response.headers.get('content-length')
+    filename = parse_requests_response(response).filename_unsafe
```

Comment:
As I said in https://github.com/marco-c/autowebcompat/pull/48#issuecomment-365648565, this needs to be sanitized.